### ISSUE TITLE
IE11: Dropdown re-opens after selecting item

### ISSUE THREAD

**Issue by sebastianhaberey** [Choices-js/Choices#621]
Happens randomly but frequently. Easy to replicate. Recorded directly from https://joshuajohnson.co.uk/Choices/:

![test](https://user-images.githubusercontent.com/4152849/62702781-fe819580-b9e7-11e9-85c7-645a04a05009.gif)

Where should we start looking for the cause? Any pointers? Regards,

Sebastian

**Comment by sebastianhaberey** [comment]
After some debugging I found that _onFocus() seems to be the cause here. After the item is selected by the user, the dropdown is hidden. Then, there's an _onFocus() call which in turn calls showDropdown().

I don't know why _onFocus() would be called here. My guess is that the component loses focus for an instant and gains it again - some IE quirk maybe?

We could probably go ahead and disable the "show dropdown on focus" functionality programmatically (we don't really need it), but if anyone can think of a better, more general idea, that'd be great!

**Comment by sebastianhaberey** [comment]
The root cause seems to be a bugfix (29ab3ca55a0b1bd0bb49743bac969569479a77d1) 
that fixed an IE11 issue where clicking on a scrollbar in the dropdown would close the dropdown instantly. I guess this issue is a side effect of this bugfix.

I believe the pull request should fix the side effect by making sure the mouse is on the scrollbar as a precondition for the fix.

**Comment by hog-seruj** [comment]
Styles are brocked on IE11 as well. 
![1574447251247](https://user-images.githubusercontent.com/5249101/69450936-e2bbb880-0d66-11ea-9ff2-38f794805961.jpg)

**Comment by jshjohnson** [comment]
> Styles are brocked on IE11 as well.

This issue in particular has been resolved

**Comment by sephyzeonas** [comment]
Hello Community,
I experienced the behavior of the original post on "version": "9.0.1" on Chrome as well.
MacOS high Sierra Version 10.13.4, 
Chrome Version 81.0.4044.113 (Official Build) (64-Bit)

I recognized the behavior still just get's triggered on a regular mouse click if you click the same item which is the current chosen one. If you hold the left mouse button (mousedown) for like 0.25s longer than an actual mouse click, the dropdown is going to work like it is intended and the dropdown is closing.

I hope this information might help you to fix the issue.

Best regards
